app/views/policies/_personal_details.html.slim

```diff
@@ -2,7 +2,7 @@
   label.control-label.col-md-3 
    | Customer
   .col-md-6
-    = f.select :user_id, options_for_select(User.all.map{ |u| [u.email, u.id]}), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
+    = f.select :user_id, options_for_select(User.all.map{ |u| ["#{u.personal_info.first_name}   #{u.personal_info.last_name} #{u.email}", u.id]}), {prompt: ApplicationHelper::DEFAULT_SELECT_OPTION}, {class: "form-control select"}
```

Comment:
@GauravMakwana-BTC :As commented above, use full name instead of first_name & lastname